requirements-frozen.txt

```diff
@@ -48,5 +48,5 @@ werkzeug==2.3.8
     #   flask
     #   flask-login
     #   howitz (pyproject.toml)
-zinolib==1.0.1
+zinolib==1.0.4
```

Comment:
Something is weird with this.  `pyproject.toml` says `zinolib>=1.1.0`, so why is zinolib `1.0.4` in the frozen requirements list?

Reply:
Because we forgot to update it probably.

Reply:
With `zinolib==1.0.4` I can't even get Howitz to run, with `1.1.0` this NTIE PR seems to work just fine for me :)